cms/djangoapps/contentstore/helpers.py

```diff
@@ -324,6 +324,50 @@ def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) ->
     return new_xblock, notices
 
 
+def import_staged_content_for_library_sync(new_xblock: XBlock, lib_block: XBlock, request) -> StaticFileNotices:
+    """
+    Import a block (along with its children and any required static assets) from
+    the "staged" OLX in the user's clipboard.
+
+    Does not deal with permissions or REST stuff - do that before calling this.
+
+    Returns (1) the newly created block on success or None if the clipboard is
+    empty, and (2) a summary of changes made to static files in the destination
+    course.
+    """
+    if not content_staging_api:
+        raise RuntimeError("The required content_staging app is not installed")
+    library_sync = content_staging_api.save_xblock_to_user_library_sync(lib_block, request.user.id)
+    if not library_sync:
+        # expired/error/loading
+        return StaticFileNotices()
+
+    store = modulestore()
+    with store.bulk_operations(new_xblock.scope_ids.usage_id.context_key):
```

Comment:
```suggestion
    with store.bulk_operations(new_xblock.context_key):
```
This is the newer, simpler way to write this.

Reply:
nice. I updated the code